Monika After Story/game/script-moods.rpy

```diff
@@ -249,6 +249,30 @@ label mas_mood_angry:
     m "Remember: A happy [player] makes a happy Monika!"
     return
     
+init 5 python:
+    addEvent(Event(persistent._mas_mood_database,"mas_mood_scared",prompt="anxious",category=[store.mas_moods.TYPE_NEUTRAL],unlocked=True),eventdb=store.mas_moods.mood_db)
+
+label mas_mood_scared:
+    m 1g "[player], are you alright?"
+    m 1f "It's worrying for me to see you're suddenly getting anxious..."
```

Comment:
I would imagine when you click on a feeling, that you are _currently_ feeling that way. That being said, I suggest that this line is brought to past tense, but having it something like:
"It's worrying for me to see you anxious..."